monkey/tests/monkey_island/cc/environment/test_user_creds.py

```diff
@@ -1,37 +1,27 @@
-from unittest import TestCase
-
 from monkey_island.cc.environment.user_creds import UserCreds
 
 
-class TestUserCreds(TestCase):
-    def test_to_dict(self):
-        user_creds = UserCreds()
-        self.assertDictEqual(user_creds.to_dict(), {})
+def test_to_dict_empty_creds():
+    user_creds = UserCreds()
+    assert user_creds.to_dict() == {}
 
-        user_creds = UserCreds(username="Test")
-        self.assertDictEqual(user_creds.to_dict(), {"user": "Test"})
 
-        user_creds = UserCreds(password_hash="abc1231234")
-        self.assertDictEqual(user_creds.to_dict(), {"password_hash": "abc1231234"})
+def test_to_dict_full_creds():
+    user_creds = UserCreds(username="Test", password_hash="abc1231234")
+    assert user_creds.to_dict() == {"user": "Test", "password_hash": "abc1231234"}
 
-        user_creds = UserCreds(username="Test", password_hash="abc1231234")
-        self.assertDictEqual(user_creds.to_dict(), {"user": "Test", "password_hash": "abc1231234"})
 
-    def test_to_auth_user(self):
-        user_creds = UserCreds(username="Test", password_hash="abc1231234")
-        auth_user = user_creds.to_auth_user()
-        self.assertEqual(auth_user.id, 1)
-        self.assertEqual(auth_user.username, "Test")
-        self.assertEqual(auth_user.secret, "abc1231234")
+def test_to_auth_user_full_credentials():
+    user_creds = UserCreds(username="Test", password_hash="abc1231234")
+    auth_user = user_creds.to_auth_user()
+    assert auth_user.id == 1
+    assert auth_user.username == "Test"
+    assert auth_user.secret == "abc1231234"
 
-        user_creds = UserCreds(username="Test")
-        auth_user = user_creds.to_auth_user()
-        self.assertEqual(auth_user.id, 1)
-        self.assertEqual(auth_user.username, "Test")
-        self.assertEqual(auth_user.secret, "")
 
-        user_creds = UserCreds(password_hash="abc1231234")
-        auth_user = user_creds.to_auth_user()
-        self.assertEqual(auth_user.id, 1)
-        self.assertEqual(auth_user.username, "")
-        self.assertEqual(auth_user.secret, "abc1231234")
```

Comment:
It looks like these tests have been removed. Do we ever use (or plan to use) the `UserCreds` class without passing it a username and password? If it will be used this way, I think we should be testing that the constructor behaves the way it should; this is part of the interface.

If `UserCreds` will never be instantiated this way, then I say we remove the default parameters from the constructor. We can do it in a later PR as long as you add a TODO or github issue so we don't forget.